samples/bot-formatting-cards/nodejs/resources/adaptiveCard.json

```diff
@@ -0,0 +1,29 @@
+﻿{
+  "$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0",
```

Comment:
Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/689/files#diff-824fbd3defb344889effcfecba9c5a1ae303b488e565c47c32494597545bba6b

Reply:
Fixed